### ISSUE TITLE
Enable OMCSessionZMQ to connect to an already running omc

### ISSUE THREAD

**Issue by bilderbuchi** [OpenModelica/OMPython#148]
As a user, I would like to be able to connect to an already running omc --interactive process that is listening at a certain host/port. 
That is, I want `OMCSessionZMQ` to *not* start a separate omc process [here](https://github.com/OpenModelica/OMPython/blob/7981c3deda64a94b0081a73eb0d8e6399d5187f5/OMPython/__init__.py#L705) if I pass in an endpoint, URI, socket/port that defines where my omc is listening.

This would be useful to enable OMPython to use an omc instance that is running elsewhere, e.g. in a managed container (that I can't start myself, so the present docker support does not help here) or on a (powerful) remote server.

**Comment by bilderbuchi** [comment]
Sounds good! These restrictions would be OK. 
I imagine I would independently start `omc --interactive --interactivePort 666` and then in Python do 
`session = OMCSessionZMQ(port=666)` (could reuse the existing `port` argument), 
`session = OMCSessionZMQ(uri='localhost:666')` or 
`session = OMCSessionZMQ(uri='\\SOMEMACHINE:666')` etc.

**Comment by arun3688** [comment]
@bilderbuchi , I guess we could use the existing port argument to connect to an existing omc connection, I can add that feature

**Comment by bilderbuchi** [comment]
@arun3688 great! I think that an additional `host` argument (defaulting to `localhost`) will be useful to pair with `host`, then one can easily assemble a proper connection string.

**Comment by scottlittle** [comment]
I would also like this feature. I am using a Mac host with a custom docker container with OpenModelica programs installed. I am able to connect to the container with a ZMQ session, but I am not able to use OMPython because of this.

**Comment by syntron** [comment]
See PR #295 ;-)

**Comment by adeas31** [comment]
@syntron this is supported now, right?

**Comment by syntron** [comment]
@adeas31 yes it is supported - see OMCSession.OMCProcessPort or as example: test_ZMQ.test_omcprocessport_execute()
